Client/app/styles/teaserView.scss

```diff
@@ -5,11 +5,12 @@
   -webkit-tap-highlight-color: transparent;
 }
 .teaser-container {
+  background:rgba(0,0,0,0.6);
   background: url("../images/background.jpg") center no-repeat fixed;
```

Comment:
either do it like 
background-color: rgba(0,0,0,0.6)
or 
background:  rgba(0,0,0,0.6) url("../images/background.jpg") center no-repeat fixed;